src/include/pbs_python.h

```diff
@@ -138,7 +138,11 @@ typedef struct	hook_input_param {
 	char		*env;
 	pbs_list_head	*jobs_list;
 	pbs_list_head	*vns_list;
+#ifndef WIN32
```

Comment:
So for WIN32 you dont have pid_t defined, right? In that can you could typedef pid_t int; for the WIN32 case, no?